packages/etherlime/cli-commands/compiler/etherlime-contract-sources/index.js

```diff
@@ -19,20 +19,30 @@ let find_contracts = async (workingPath) => {
     }
 
     try {
-      let files = [];
-      const readFiles = await fs.readdirSync(workingPath);
-      for (let i = 0; i < readFiles.length; i++) {
-        let currentPath = path.join(workingPath, readFiles[i]);
-        files.push(currentPath)
-      }
-      return resolve(fetchSolAndVyperFiles(files))
+      const readFiles = await readAllFilesSync(workingPath);
+      return resolve(fetchSolAndVyperFiles(readFiles))
     } catch (err) {
       return reject(err)
     }
 
   });
 }
 
+const readAllFilesSync = async function(dir, fileList) {
+  files = await fs.readdirSync(dir);
+  fileList = fileList || [];
+  files.forEach(async function(file) {
```

Comment:
George doesn't like `forEach` to be used :)

Reply:
Yep, you are right, It shouldn't work properly with async/await